src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -55,9 +55,9 @@ public function new(Request $request, EntityManagerInterface $entityManager): Re
 <?php } ?>
 
 <?php if ($use_render_form) { ?>
-        return $this->renderForm('<?= $templates_path ?>/new.html.twig', [
+        return $this->render('<?= $templates_path ?>/new.html.twig', [
```

Comment:
We'll need to do this conditionally. E.g. if Symfony >6.1 `render()` else `renderForm()`

It looks like in 6.2+, the `render()` method is able to set the appropriate response based on `FormInterface` parameters. (`isSubmitted()` && `isValid()`). As far as I know, that logic does _not_ exist in `render()` in `5.4.`.

Adding a test to ensure this logic works correctly regardless of the version would be awesome!

Reply:
I will update my PR soon :D